### ISSUE TITLE
Formatting Code

### ISSUE THREAD

**Issue by samuelbelko** [jbrea/BayesianOptimization.jl#29]
Hi,
are there any opinions on code formatting to make this project look more consistent and readable? I would suggest to use [the formatting feature ](https://www.julia-vscode.org/docs/stable/userguide/formatter/#Formatting-Code) of VSCode with Julia extention. This seems to be quite unintrusive and convenient.
Thanks!

**Comment by jbrea** [comment]
Yes, we can do this. I think the vscode plugin is based on JuliaFormatter that could also be used outside of vscode, e.g as a gh action triggered by PRs. But I guess it would be sufficient to just make sure every contributor uses it.

**Comment by samuelbelko** [comment]
Good, thanks 👍
